make_asym.c

```diff
@@ -126,14 +126,19 @@ PyInit_make_asym(void)
 #else
 #define INITERROR return
 
-void
+#if PY_MAJOR_VERSION >= 3
+#define NUMPY_IMPORT_ARRAY_RETURN_TYPE int
+#else
+#define NUMPY_IMPORT_ARRAY_RETURN_TYPE void
+
+#endif
+
+NUMPY_IMPORT_ARRAY_RETURN_TYPE
 initmake_asym(void)
 {
-  import_array();
-  return Py_InitModule("make_asym", make_asym_methods);
+Py_InitModule("make_asym", make_asym_methods);
+import_array();
 }
-#endif
```

Comment:
I thought that this was covered by adding an `#endif` exactly below the #define statement.  When I compile this code, I do not receive an error at this line. 

I will now add the `#endif` back into the code segment, but if it results in future errors, we know where to look.